examples/example.cpp

```diff
@@ -47,7 +47,7 @@ int main(const int argc, const char* argv[])
   auto chemical_system = solver_params.system_;
   auto reactions = solver_params.processes_;
 
-  auto solver = CpuSolverBuilder<micm::RosenbrockSolverParameters>(solver_params.parameters_)
+  auto solver = CpuSolverBuilder_DoolittleLU<micm::RosenbrockSolverParameters>(solver_params.parameters_)
```

Comment:
```suggestion
  auto solver = DefaultCPUSolverBuilder<micm::RosenbrockSolverParameters>(solver_params.parameters_)
```
I prefer this instead of adding doolittle to the name. What do you think?

Reply:
agreed, or just leave it as `CpuSolverBuilder`. If the MOZART LU algorithm is faster, we'll probably just use that in the examples and tutorials

Reply:
As Matt suggested, we may just keep the original name for now.